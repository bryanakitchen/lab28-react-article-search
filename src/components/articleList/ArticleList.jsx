import React from 'react';
import Article from '../article/Article';

export default function ArticleList() {
  return (
    <>
      <div>
          Hello from the ArticleList component.
      </div>
      <Article />
    </> 
  );
}
