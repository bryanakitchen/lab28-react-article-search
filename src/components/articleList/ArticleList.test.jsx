import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ArticleList from './ArticleList';

describe('ArticleList component', () => {
  afterEach(() => cleanup());
  it('renders ArticleList', () => {
    const { asFragment } = render(<ArticleList
      articles={[]}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});

// {
//     title: 'Happy Trees',
//     author: 'Bob Ross',
//     description: 'this article talks about drawing happy trees'
//   },
//   {
//     title: 'Happy Trees 2',
//     author: 'Bob Ross',
//     description: 'this article talks about drawing more happy trees'
//   },
