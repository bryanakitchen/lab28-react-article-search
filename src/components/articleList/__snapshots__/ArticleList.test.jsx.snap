// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList component renders ArticleList 1`] = `
<DocumentFragment>
  <ul
    data-testid="articles"
  >
    <li>
      <h2>
        Happy Trees
      </h2>
      <h3>
        Bob Ross
      </h3>
      <p>
        this article talks about drawing happy trees
      </p>
    </li>
    <li>
      <h2>
        Happy Trees 2
      </h2>
      <h3>
        Bob Ross
      </h3>
      <p>
        this article talks about drawing more happy trees
      </p>
    </li>
  </ul>
</DocumentFragment>
`;
